template-parts/content/content-none.php

```diff
@@ -20,16 +20,12 @@
 		if ( is_home() && current_user_can( 'publish_posts' ) ) :
 
 			printf(
-				'<p>' . wp_kses(
-					/* translators: 1: link to WP admin new post page. */
-					__( 'Ready to publish your first post? <a href="%1$s">Get started here</a>.', 'twentynineteen' ),
-					array(
-						'a' => array(
-							'href' => array(),
-						),
-					)
-				) . '</p>',
-				esc_url( admin_url( 'post-new.php' ) )
+				'<p>' .
+					/* translators: %s: link to WP admin new post page. */
+					esc_html__( 'Ready to publish your first post? %1$sGet started here%2$s.', 'twentynineteen' )
+				. '</p>',
+				'<a href="' . esc_url( admin_url( 'post-new.php' ) ) . '">',
+				'</a>'
```

Comment:
This change doesn't really make the code more readable. Also, the translators comment is not correct since there is no `%s` placeholder.

Worth to mention that `Ready to publish your first post? <a href="%1$s">Get started here</a>.` is a string that's already used by many themes.